src/utils/string.hpp

```diff
@@ -301,6 +301,26 @@ inline std::vector<std::string> RSplit(const std::string_view src, const std::st
   return res;
 }
 
+/**
+ * Get a first part of a string by `delimiter` into a string.
+ */
+inline std::string_view GetFirstPartOfSplit(const std::string_view src, const char delimiter) {
+  auto delimPos = src.find(delimiter);
+  return delimPos == std::string::npos ? src : src.substr(0, delimPos);
```

Comment:
Yes, I thought about this. Will check performance, because we have one place `DeserializePropertiesFromAuxiliaryStorages` we pass string_view to function. But there should not be degradation. Will check

Reply:
how, `If no such character was found, there are no tokens in str at all, and the function returns a null pointer`?

Reply:
In this case that means that string contains only delimiter/s.